cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -331,13 +331,14 @@ static Bom addFormulation(final String fileName, final String name) {
         Bom bom = readJSONfile(fileName);
         List<Formula> formulation = bom.getFormulation();
         if (formulation == null) {
-          formulation = new LinkedList<Formula>();
-          Formula formula  = new Formula();
-          System.err.println("SXAECW: " + name);
-          formula.setBomRef(name);
-          formulation.add(formula);
-          bom.setFormulation(formulation);
+            formulation = new LinkedList<Formula>();
+            bom.setFormulation(formulation);
         }
+        Formula formula = new Formula();
+        System.err.println("SXAECW: " + name);
```

Comment:
This line can be removed as it was just for debug :-)
I could do it in a separate PR but it might be easier to just remove it in yours to avoid a merge conflict.

Reply:
Sure, I fix it :)